code/modules/awaymissions/mission_code/Academy.dm

```diff
@@ -172,6 +172,10 @@
 	var/used = FALSE
 	var/roll_in_progress = FALSE
 
+/obj/item/dice/d20/fate/proc/crystalrefresh()
+	sides = 19
+	visible_message("<span class='danger'>The 20th face of the [src] falls off and crumbles to dust!</span>")
```

Comment:
```suggestion
/obj/item/dice/d20/fate/proc/crystalrefresh()
	if(sides == 20)
		sides = 19
		visible_message("<span class='danger'>The 20th face of the [src] falls off and crumbles to dust!</span>")
```

needs a check to ensure the 20th face doesn't crumble off more than once, no?

Reply:
That way wouldn't work, changed it in a way that does.